packages/react/src/components/Tooltip/index.tsx

```diff
@@ -143,35 +138,35 @@ export default function Tooltip({
     return () => {
       targetElement.removeEventListener('keyup', handleEscape);
     };
-  }, [showProp]);
+  }, [showTooltip]);
```

Comment:
This was a bug--`showProp` doesn't currently drive the visibility of the tooltip. As you can see on line 132, this effect is checking `showTooltip`